Frameworks/QuartzCore/CALayer.mm

```diff
@@ -623,8 +623,6 @@ - (void)display {
             CGContextTranslateCTM(drawContext, 0, float(target->Backing()->Height()));
         }
         if (priv->contentsScale != 1.0f) {
-            CGContextScaleCTM(drawContext, priv->contentsScale, priv->contentsScale);
-
```

Comment:
This scale is for Cairo's sake and needs to be done.
Perhaps we should keep it and apply an inverse scale based on the DPI to the D2D context instead of eliminating it here.